rlcs.user.js

```diff
@@ -763,8 +857,9 @@ ________________________________________________________________________________
 					// Narration Style
 					var msg;
                     var usr = $usr.text();
-                    if (usr == "741456963789852123") { usr = "7-41"; }
-                    //RLClog("usr name for narration "+usr)
+                    if (usr == "741456963789852123") { usr = "7 4 1"; } /* idea: if username is a lot of numbers, call them by the first 3 numbers seperated */
```

Comment:
I didn't, it was already there, Stjerneklar put it there, the version was wrong, forked on 3.5.2 but 3.8.3(which I actually forked from) already had it